_includes/footer.html

```diff
@@ -0,0 +1,51 @@
+<!-- create a footer -->
+<footer class="footer">
+  <div class="row mx-0">
+    <div class="col-md-5 left-div">
+      <a href="{{ '/' | relative_url }}">
+        <div class="footer-logo">
+          <img src="./assets/img/mlh-prep.png" />
+        </div>
+      </a>
+
+      <div class="social-links">
+        <a href="https://www.linkedin.com/school/mlh-fellowship/" class="links"
+          ><i class="fab fa-linkedin fa-lg"></i
+        ></a>
+        <a href="https://www.instagram.com/mlhacks/" class="links"
+          ><i class="fab fa-instagram fa-lg"></i
+        ></a>
+        <a
+          href="https://twitter.com/MLHacks?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor"
+          class="links"
+          ><i class="fab fa-twitter fa-lg"></i
+        ></a>
+        <a href="https://www.facebook.com/MajorLeagueHacking" class="links"
+          ><i class="fab fa-facebook fa-lg"></i
+        ></a>
+        <a href="https://www.youtube.com/c/Majorleaguehacking" class="links"
+          ><i class="fab fa-youtube fa-lg"></i
+        ></a>
+      </div>
+    </div>
+
+    <div class="col-md-6 col-12 right-div">
+      <div class="list">
+        <p>
+          <a href="/#">Blog</a>
+        </p>
+        <p>
+          <a href="/#about">About</a>
+        </p>
+        <p>
+          <a href="/#">Social</a>
```

Comment:
I don't think Social needs to be there. You can add the subsections which are there in the project so, meet the pod, projects and blog(which will be added soon). Let me know what you think

Reply:
Yes

Reply:
Sorry, i made a mistake... Meet the pod will point to the description, so I guess there's no need for an About section